Description: Compat with gnutls 3.6.4, just ask for normal priority.
 otherwise ssl connections all fail.
Author: Dimitri John Ledkov <xnox@ubuntu.com>

--- elinks-0.12~pre6.orig/src/network/ssl/ssl.c
+++ elinks-0.12~pre6/src/network/ssl/ssl.c
@@ -256,7 +256,7 @@ init_ssl_connection(struct socket *socke
 	}
 
 #ifdef HAVE_GNUTLS_PRIORITY_SET_DIRECT
-	if (gnutls_priority_set_direct(*state, "NORMAL:-CTYPE-OPENPGP", NULL)) {
+	if (gnutls_priority_set_direct(*state, "NORMAL", NULL)) {
 		gnutls_deinit(*state);
 		mem_free(state);
 		return S_SSL_ERROR;
